openlibrary/templates/books/add.html

```diff
@@ -71,7 +71,7 @@ <h1>$_("Add a book to Open Library")</h1>
         $if recaptcha and not ctx.user:
             <div class="formElement">
                 <div class="label">$_("Since you are not logged in, please type in the text or number(s) below.")</div>
-                $:recaptcha.render()
+                <div class="g-recaptcha" data-sitekey="$:recaptcha.public_key"></div>
```

Comment:
Has this been tested to make sure the `data-sitekey` is correct? I wonder if this obj has a `public_key` value

Reply:
tested -- this looks right